src/styles/header.css

```
@@ -0,0 +1,48 @@
+.header-menu {
+  text-align: left;
+  height: 85px;
+  padding: 10px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.header-menu > .logo > img {
+  display: block;
+  margin: 0;
+  width: 10em;
+}
+
+.color-text-header {
+  color: var(--my-color, #2c2c51);
```

vc não entendeu o que é variável, isso vc declara no `base.css` e atribui a  uma variável aí aqui vc só referencia
```suggestion
  color: var(--my-var);
```

---

https://www.w3schools.com/css/tryit.asp?filename=trycss3_var

---

Agora consegui entender